Doc/howto/clinic.rst

```diff
@@ -310,17 +324,21 @@ Let's dive in!
 
    Sample::
 
-        /*[clinic input]
-        module _pickle
-        class _pickle.Pickler "PicklerObject *" "&Pickler_Type"
-        [clinic start generated code]*/
+       /*[clinic input]
+       class _asyncio.Future "FutureObj *" "&Future_Type"
+       [clinic start generated code]*/
 
-        /*[clinic input]
-        _pickle.Pickler.dump
+       /*[clinic input]
+       _asyncio.Future.add_done_callback
+
+           cls: defining_class
+           fn: object
```

Comment:
This breaks with the text above:

> For now you're going to use what's called a "legacy converter"

Changing this means changing the course of the tutorial, which IMO is a bit too drastic. Can you find another example which retains the overall course of the tutorial?

Reply:
Well, I know `defining_class` is the advance usage. What if removing this line? Otherwise, I will take time to find another function.

Reply:
Sorry for not expressing myself clearly: what I meant was that the previous example used the legacy `'O'` converter instead of the modern `object` converter.

Also, as you point out, `defining_class` is for advanced use, so it is not a good fit for the tutorial.


Alternatively, we can just make up an example; it does not have to be something from the stdlib :)